vmdk_plugin/utils/fs/fs.go

```diff
@@ -37,6 +37,9 @@ const pciAddrLen = 10                     // Length of PCI dev addr
 // FstypeDefault contains the default FS when not specified by the user
 const FstypeDefault = "ext4"
 
+// BinSearchPaths contains search paths for host binaries
+var BinSearchPaths = [4]string{"/bin", "/sbin", "/usr/bin", "/usr/sbin"}
```

Comment:
Even if we want to have a list here maybe a slice can be used vs. an array - []string{"/bin", "/sbin", "/usr/bin", "/usr/sbin"}